rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -73,9 +65,7 @@ private FlowNodeUtil() {
     }
 
     public abstract static class CacheExtensionPoint implements ExtensionPoint {
-        public abstract Cache<String,List<FlowNode>> getExecutionCache();
```

Comment:
We're not really following SemVer for plugins or Jenkins as a whole properly anyway, but true. 

This is an extension point *only* to resolve issues with JenkinsRule seeing stale cache data in tests, and has no consumers or implementations but this one, so an :ant: at best.